sdk/python/examples/quickstart/Quickstart.ipynb

```diff
@@ -0,0 +1,239 @@
+    "                           staging_location=STAGING_LOCATION,\n",
+    "                           id_column='ride', \n",
+    "                           timestamp_column='pickup_datetime')\n",
+    "\n",
+    "# Update feature and entity metadata\n",
+    "serving_ds=Datastore(id='REDIS1').spec\n",
+    "warehouse_ds=Datastore(id='NOOP').spec\n",
+    "\n",
+    "# Update feature and entity metadata. Ideally you want to update these manually\n",
+    "# so that they contain adequate information for the next user\n",
+    "importer.entity.description = 'nyc taxi dataset' \n",
+    "for feature in importer.features:\n",
+    "    feature.serving_store = serving_ds\n",
+    "    feature.warehouse_store = warehouse_ds\n",
+    "    feature.description = 'nyc taxi dataset'\n",
+    "    \n",
+    "# Ingest the feature data into the store\n",
+    "fs.run(importer, apply_features=True, apply_entity=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating a training dataset\n",
+    "\n",
+    "Creating a training dataset allows you to isolate the data that goes into the model training step, allowing for reproduction and traceability."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Retrieving data: Training\n",
+    "\n",
+    "feature_set = FeatureSet(entity=\"ride\", \n",
+    "                         features=[\"ride.none.log_trip_duration\", \n",
+    "                                  \"ride.none.distance_haversine\",\n",
+    "                                  \"ride.none.distance_dummy_manhattan\",\n",
+    "                                  \"ride.none.direction\",\n",
+    "                                  \"ride.none.month\",\n",
+    "                                  \"ride.none.day_of_month\",\n",
+    "                                  \"ride.none.hour\",\n",
+    "                                  \"ride.none.day_of_week\",\n",
+    "                                  \"ride.none.vi_1\",\n",
+    "                                  \"ride.none.vi_2\",\n",
+    "                                  \"ride.none.sf_n\",\n",
+    "                                  \"ride.none.sf_y\"])\n",
+    "dataset_info = fs.create_training_dataset(feature_set, \"2016-06-01\", \"2016-08-01\")\n",
+    "dataset = fs.download_dataset_to_df(dataset_info, staging_location=STAGING_LOCATION)\n",
+    "\n",
+    "dataset.head()\n",
+    "\n",
+    "# train your model\n",
+    "# ..."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Retrieving serving data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Retrieving data: Serving\n",
+    "\n",
+    "# set serving endpoint\n",
+    "fs.serving_url = FEAST_SERVING_URL\n",
```

Comment:
Can't this be set at the start during client creation?

Reply:
yes, but i dont think people would use core and serving within the same session so it makes sense to allow users to set it separately

Reply:
What if you just create a new client?

Reply:
Yeah, seems good. Done.